annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -782,11 +793,6 @@ private String getTumorSeqAllele(MutationRecord record) {
         }
     }
 
-    @Override
-    public boolean isHgvspNullClassifications(String variantClassification) {
-        return hgvspNullClassifications.contains(variantClassification);
-    }
-
     private static List<String> initNullClassifications() {
```

Comment:
I would move this up to where it is used.  More importantly, we should figure out a better way to keep this list in sync/shareable with [MutationFilter.java](https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/java/org/mskcc/cbio/portal/scripts/MutationFilter.java)